dev-perl/archive_extract/archive_extract-0.88.recipe

```diff
@@ -38,5 +39,10 @@ BUILD()
 
 INSTALL()
 {
-	make install DESTDIR="${DESTDIR}"
+	make install
+
+	# remove architecture-specific files
+	cd $prefix
+	rm -r $(perl '-V:archlib|vendorarch' | cut -d\' -f2 | cut -d/ -f5-)
+		# cut extracts the quoted string and strips the prefix (which is perl's and not ours)
 }
```

Comment:
This is a problem with compress_bzip2

Reply:
This is only meant for "any" arch recipes, not for architecture specific ones.